src/edu/csus/ecs/pc2/core/model/InternalContest.java

```diff
@@ -1493,7 +1493,7 @@ public Run checkoutRun(Run run, ClientId whoChangedRun, boolean reCheckoutRun, b
                 runList.updateRun(newRun);
                 return runList.get(run.getElementId());
             } else {
-                throw new RunUnavailableException("Client " + clientId + " can not checked out run " + run.getNumber() + " (site " + run.getSiteNumber() + ")");
+                throw new RunUnavailableException("Client " + whoChangedRun + " can not check out run " + run.getNumber() + " (site " + run.getSiteNumber() + ")");
```

Comment:
As long as we're fixing typos ("checked" -> "check") I'd like to see "can not" corrected to "cannot".  ("Can not" is properly used when the "not" is part of a separate phrase -- for example, "I can check out the run, or I can not check out the run; you tell me which you prefer I should do".  There's no separate phrase here to which "not" applies.)
But this is obviously totally trivial in terms of the real issue -- the wrong variable being printed.